config/software/omnibus-toolchain.rb

```diff
@@ -28,6 +28,9 @@
 # git!
 dependency "git"
 
+# fakeroot for EL based things
+dependency "fakeroot" if el?
```

Comment:
Yeah the only reason that conditional existed was the `fakeroot` package only existed for EL. Now that we are compiling from source we should make it available to all platforms.